src/org/labkey/test/tests/FieldEditorAddRowSelectionActionTest.java

```diff
@@ -0,0 +1,215 @@
+package org.labkey.test.tests;
+
+import org.jetbrains.annotations.Nullable;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.labkey.remoteapi.domain.PropertyDescriptor;
+import org.labkey.serverapi.reader.Readers;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.components.DomainDesignerPage;
+import org.labkey.test.components.domain.DomainFormPanel;
+import org.labkey.test.pages.ReactAssayDesignerPage;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.IssuesHelper;
+import org.labkey.test.util.PortalHelper;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FieldEditorAddRowSelectionActionTest extends BaseWebDriverTest
+{
+    private final static String PROJECT_NAME = "Field Editor Add Row Selection Action Test";
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        FieldEditorAddRowSelectionActionTest init = (FieldEditorAddRowSelectionActionTest) getCurrentTest();
+        init.doSetup();
+    }
+
+    @Override
+    protected @Nullable String getProjectName()
+    {
+        return PROJECT_NAME;
+    }
+
+    @Override
+    public List<String> getAssociatedModules()
+    {
+        return Arrays.asList("study");
+    }
+
+    @Override
+    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        super.doCleanup(afterTest);
```

Comment:
If all you are doing in the doCleanup is calling super.doCleanup you shouldn't need to have this override.

Reply:
Removed